### ISSUE TITLE
Tutorial: marking step cannot accept marker clicks

### ISSUE THREAD

**Issue by drphilmarshall** [zooniverse/Lens-Zoo#290]
I logged out so that I could test the tutorial. It looks OK, but fails on the part that says "Marking Lensed Features. If you see something that is being lensed, mark it. In this case, click on the arc." I am unable to place a marker, and so am unable to advance through the tutorial. Here's the js error:
```
start image		 ASW0002oqg
Uncaught SecurityError: Failed to execute 'getImageData' on 'CanvasRenderingContext2D': The canvas has been tainted by cross-origin data.
```
This is a show-stopper!

**Comment by chrissnyder** [comment]
That will only occur in local testing. The domains match in production and it doesn't throw that security warning.

**Comment by drphilmarshall** [comment]
Great! Thanks Chris.

**Comment by drphilmarshall** [comment]
OK, I see the marker appear in the tutorial, but that does not cause a blue "continue" box to appear - the tutorial just stops.

**Comment by chrissnyder** [comment]
Closed via 281f738d2879ca2e8508905079379f745789ed60

**Comment by aprajita** [comment]
Hi @chrissnyder - Reopening as when I mark the lensed arc, i get "Whoops, try again..." message even though my marker is on the arc. Perhaps something is checking the wrong co-ordinate mask (perhaps another VICS82/CFHTLS difference?). 

This message also means I don't see continue and the tutorial just stops as @drphilmarshall mentioned.

**Comment by chrissnyder** [comment]
Does the browser console output anything?

**Comment by aprajita** [comment]
no, I don't see anything in the browser console, when I click and the message comes up.

**Comment by chrissnyder** [comment]
What browser and could you post a screenshot?

**Comment by aprajita** [comment]
Firefox 40.0

![swerra](https://cloud.githubusercontent.com/assets/1917970/9993450/c9e2f088-606f-11e5-9624-8977721f2e0d.png)

I just tried it in Safari v6.2.3 and didn't get this, i.e. got "Great job..." message with continue button.

**Comment by chrissnyder** [comment]
How about now?

**Comment by aprajita** [comment]
hi, tried on firefox 40.0 and 40.0.3 and it's the same "whoops" message

**Comment by chrissnyder** [comment]
Did some more work, looks to work in the browsers I could test.

**Comment by aprajita** [comment]
Ok don't worry about this, I doubt it will affect many.
thanks

**Comment by aprajita** [comment]
think this has been fixed by @chrissnyder work relating to #293, so closing this now.
